website/apps/styles.css

```diff
@@ -73,4 +118,215 @@
 
 #footer a {
   padding: .5em;
+}
+
+/*
+  -------
+  Buttons 
+  -------
+*/
+
+button, .button {
+  text-align: center;
+  margin: 0;
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
+
+button.ghost, .button.ghost {
+  border: 1px solid var(--accent);
+  background: var(--bg);
+  color: var(--accent);
+}
+
+/*
+  -------
+  Dialogs 
+  -------
+*/
+
+.dialog-backdrop {
+  background: rgb(0 0 0 / 30%);
+  display: none;
+  position: fixed;
+  overflow-y: auto;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 1;
+}
+
+.dialog-backdrop.active {
+  display: block;
+}
+
+[role=dialog] {
+  font-size: 1rem;
+  display: none;
+  padding: 2rem;
+  background: var(--bg);
+  min-height: 100vh;
+  gap: 1rem;
+}
+
+[role=dialog] .app-container {
+  display: flex;
+  gap: var(--padding);
+}
+
+[role=dialog] .button-container {
+  display: flex;
+  gap: var(--padding);
+  flex-direction: column;
+}
+
+[role=dialog].active {
+  display: flex;
+  flex-direction: column;
+}
+
+[role=dialog] .metadata {
+  display: flex;
+  flex-direction: column;
+}
+
+[role=dialog] .additional-info {
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+}
+
+/*
+  -------
+  Tabs 
+  -------
+*/
+
+#tabs {
+  position: sticky;
+  bottom: 0;
+  padding: 1rem;
+  background: #FFFFFF90;
+  border-top: 1px solid #00000020;
+  backdrop-filter: blur(1rem);
+  display: flex;
+  gap: 1rem;
+  width: 100%;
+}
+
+#tabs > button {
+  width: 100%;
+  background: transparent;
+  border: none;
+  color: black;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-size: 1rem;
+  transition: all .3s;
+}
+
+#tabs > button:hover {
+  filter: brightness(1.9);
+  background: #00000010;
+}
+
+#tabs > button:active {
+  background: #00000010;
+}
+
+#tabs > button.active {
+  background: #00000010;
```

Comment:
maybe show the active tab a bit clearer, i.e.:

```suggestion
  background: #0080F060;
```

before / after:

<img width=320 src=https://github.com/user-attachments/assets/b2d35704-b3e9-4d43-a6f5-d2ac9041df7f>
<img width=320 src=https://github.com/user-attachments/assets/d8c8fe3d-d203-4630-bf69-f0e8522ea576>